ui/app/src/Views/SilenceForm/Updates.elm

```diff
@@ -172,5 +191,17 @@ update msg model =
 
                 newSilence =
                     toSilence newForm
+
+                -- In case the new silence is valid, update the affected alerts list
+                cmd =
+                    if isFormValid newForm then
+                        case newSilence of
+                            Ok silence ->
+                                Task.perform (PreviewSilence >> MsgForSilenceForm) (Task.succeed silence)
```

Comment:
I think that's what's happening.

I think for now we should keep the preview a manual feature, and create a new issue for adding a debounce on keystrokes and refreshing after, e.g., a 500ms delay since the last keystroke (if the form is valid)